server/sonar-server/src/main/java/org/sonar/server/metric/ws/DeleteAction.java

```diff
@@ -64,15 +62,13 @@ public void define(WebService.NewController context) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
-    userSession.checkLoggedIn().checkPermission(GlobalPermissions.SYSTEM_ADMIN);
-    DbSession dbSession = dbClient.openSession(false);
-    try {
+    userSession.checkLoggedIn().checkIsRoot();
```

Comment:
description should be fixed, root is now required, not system administration... or is that the name we give to root?